src/core/triggers.systemd.in

```diff
@@ -69,3 +70,90 @@ if posix.access("%{_localstatedir}/lib/rpm-state/systemd/needs-reload") then
         posix.wait(pid)
     end
 end
+
+%transfiletriggerin -P 100700 -p <lua> -- @sysusersdir@
```

Comment:
please leave sysusers out for now, we need to sort it out in a different way, probably.

Reply:
Why? We've been using this for the entire time Mageia has been using systemd (since Mageia 2).

It works just fine. The only thing I can think of is that maybe it should be `%filetriggerin` instead of `%transfiletriggerin`.